jupyterlab_gitplus/handlers.py

```diff
@@ -18,13 +18,15 @@
 
 GITHUB_ENDPOINT = 'https://github.com/'
 GITHUB_REMOTE_DOMAIN = 'github.com'
-REVIEWNB_ENDPOINT = 'https://app.reviewnb.com/'
 
 
 class ModifiedRepositoryListHandler(IPythonHandler):
     '''
     Given a list of recently opened files we return repositories to which these files belong to (if the file is under a git repository)
     '''
+    def initialize(self, **kwargs):
```

Comment:
Is this required now?

Since we're not doing anything here wouldn't the base `initialize` method (from  `IPythonHandler`) be enough?

Reply:
No, because the base method doesn't take `**kwargs` and we pass them in.